src/components/Navbar/Navbar.jsx

```diff
@@ -18,8 +18,10 @@ export function Navbar(){
         </nav>
         <nav className="navigation">
             <ul>
-                <li><Link to="/explore" className="btn btn-secondary-link">Explore</Link></li>
+                <li><Link to="/videos" className="btn btn-secondary-link">Explore</Link></li>
```

Comment:
As these list items have same styling, so you can map over an array and can return these list.

Like,
```javascript
[{linkTo: "/videos", linkFor: "Explore"}, ...other objects].map(({linkTo, linkFor}) => <li 
to={linkTo} className="...">{linkFor}</li>)
```

Reply:
Thanks for the suggestion @hsnice16 , I will update it.